sanic/testing.py

```diff
@@ -26,6 +26,7 @@ def __init__(self, app):
                     session, method.lower())(url, *args, **kwargs) as response:
                 try:
                     response.text = await response.text()
+                    response.json = await response.json()
```

Comment:
@messense - the code can't really use await response.json() when using the test_client since it'll complain that you can't run the client while another loop is running. So I don't think no one is using it at this point.

#771 - related.